Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -324,7 +324,9 @@ public List<Result> ContextMenus(IPublicAPI api)
 
                         Action = _ =>
                         {
-                            Main.StartProcess(Process.Start, new ProcessStartInfo(Package.Location));
+                            Main.StartProcess(Process.Start, new ProcessStartInfo(
+                            !string.IsNullOrEmpty(Main._settings.CustomizedExplorer) ? Main._settings.CustomizedExplorer:Settings.Explorer,
+                            $"{Main._settings.CustomizedArgs} \"{Package.Location}\"".Trim()));
```

Comment:
does UWP work with Args? Since "/select,%f" is now the default init

Reply:
Well...That's an issue... Let me take a look on solution.

Reply:
i got it working by removing CustomizedAgs, if it doesnt work at all for UWP, perhaps remove it?